.gitlab-ci.yml

```diff
@@ -78,6 +78,19 @@ test-linux-stable:                 &test
     - time cargo test --all --release --verbose --locked
     - sccache -s
 
+check-web-wasm:
+  stage:                           test
+  image:                           tomaka/cargo-web:latest
+  allow_failure: true
+  only:
+    - master
```

Comment:
this test will only be run after pull requests already have been merged. is that intended?

Reply:
No!

Reply:
> whole point of this CI step is to detect when the build breaks.

In current config it will run during nightly builds and after the PR merges. It won't break the pipeline, but in case of failure you'll see a yellow "!" in gitlab's pipeline. And won't be notified by chat or email.

So if you want to run it each commit in each PR, you need to remove `only: master`.
If you want it to break the pipeline, remove `allow_failure: true`.
Alternatively, it can be launched independently, on any cron schedule or even a webhook.